### DISCUSSION TITLE
23.07

### DISCUSSION THREAD

**Discussion by falkTX** [DISTRHO/Cardinal#543] [Announcements]
Cardinal 23.07

<hr /><em>This discussion was created from the release <a href='https://github.com/DISTRHO/Cardinal/releases/tag/23.07'>23.07</a>.</em>

**Comment by hmturnbull** [comment]
Where's the Windows version?  All I see here is Linux.

**Reply by dromer** [reply]
It is still building. This is a very big project.

https://github.com/DISTRHO/Cardinal/actions/runs/5563446285

**Comment by benaldhouse** [comment]
I now have the standalone as well as the plugins in my installation. This time I managed to spot the 'stand alone' check box in the installation dialog :-)). Lovely!

**Comment by subderground** [comment]
Great work, everything is fine here on win11 and Reaper 🤩 Many thanks.

**Comment by rustfuture** [comment]
I really love Cardinal, but I can no longer use it due to this update, as my Mac version is 10.12.6. Is there a way we can resolve this situation?

**Comment by opaisa** [comment]
I've recently discovered Cardinal and have really enjoyed it so far, and was looking forward to using it's updated features. However, I'm still on OSX 10.13.6 and am unable to upgrade due to my current hardware, essentially locking me out of the newest (and I'm assuming future) releases. Is there a reason the required OS version was bumped up to 10.15? And is there a way that users of older OS's can get the new releases running on our systems? Thanks.
